internal/services/instance/types.go

```diff
@@ -66,7 +66,7 @@ func flattenImageExtraVolumes(volumes map[string]*instance.Volume, zone scw.Zone
 		volumeFlat := map[string]interface{}{
 			"id":                zonal.NewIDString(zone, volume.ID),
 			"name":              volume.Name,
-			"export_uri":        volume.ExportURI,
+			"export_uri":        volume.ExportURI, //nolint:staticcheck
```

Comment:
We need to add test and documentation to ensure that we detect when this field is removed from API responses and that the user are warned that the field is deprecated.